docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+++++
+<div class="tabs" data-tab-group="fleet-k8s">
+  <div role="tablist" aria-label="Fleet Server on Kubernetes">
+    <button role="tab"
+            aria-selected="true"
+            aria-controls="quickstart-tab"
+            id="quickstart">
+      Quick start
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="production-tab"
+            id="production"
+            tabindex="-1">
+      Production
+    </button>
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="quickstart-tab"
+       aria-labelledby="quickstart">
+++++
++
+include::add-fleet-server-kubernetes-content.asciidoc[tag=quickstart-secret]
++
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="production-tab"
+       aria-labelledby="production"
+       hidden="">
+++++
++
+include::add-fleet-server-kubernetes-content.asciidoc[tag=production-secret]
++
+++++
+  </div>
+</div>
+++++
+
+. Save the proposed `Deployment` manifest locally, for example as `fleet-server-dep.yaml`, and adapt it to your needs:
++
+++++
+<div class="tabs" data-tab-group="fleet-k8s">
+  <div role="tablist" aria-label="Fleet Server on Kubernetes">
+    <button role="tab"
+            aria-selected="true"
+            aria-controls="quickstart-dep-tab"
+            id="quickstart-dep">
+      Quick start
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="production-dep-tab"
+            id="production-dep"
+            tabindex="-1">
+      Production
+    </button>
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="quickstart-dep-tab"
+       aria-labelledby="quickstart-dep">
+++++
++
+include::add-fleet-server-kubernetes-content.asciidoc[tag=quickstart-deployment]
++
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="production-dep-tab"
+       aria-labelledby="production-dep"
+       hidden="">
+++++
++
+include::add-fleet-server-kubernetes-content.asciidoc[tag=production-deployment]
++
+++++
+  </div>
+</div>
+++++
+
+. Deploy the configured manifest to create and start the {fleet-server}:
++
+[source, shell]
+------------------------------------------------------------
+kubectl apply -f fleet-server-dep.yaml
+------------------------------------------------------------
++
+[IMPORTANT]
+====
+Ensure the `Deployment` and all referenced `Secrets` are created in the same `Namespace`.
+====
+
+. Check the {fleet-server} Pod logs for errors and confirm in {kib} that the {fleet-server} agent appears as `Connected` and `Healthy` in *{kib} → Fleet*.
++
+[source, shell]
+------------------------------------------------------------
+kubectl logs fleet-server-xxxxx
+------------------------------------------------------------
++
+It could take a while until the {fleet-server} fully starts. If you left the {kib} browser window opened during <<add-fleet-server-kubernetes-preparations>> it will show *Connected* in case everything goes well.
```

Comment:
```suggestion
It can take a couple of minutes for {fleet-server} to fully start. If you left the {kib} browser window open during <<add-fleet-server-kubernetes-preparations>> it will show *Connected* when everything has gone well.
```
I'd say "a couple of minutes" just so people know how long they should wait in case Fleet Server doesn't enroll successfully, but please change if that's not the right period of time.